apps/condo/domains/billing/schema/BillingReceipt.test.js

```diff
@@ -813,6 +813,42 @@ describe('BillingReceipt', () => {
                 expect(receipt).toHaveProperty(['recipient', 'tin'])
                 expect(receipt).toHaveProperty(['receiver', 'tin'], receipt.recipient.tin)
             })
+            test('Should create recipient and set receiver.isApproved field automatically if billing integration is trusted and tin matches', async () => {
+                const { context: trustedBillingContext, organization: trustedOrganization } = await makeContextWithOrganizationAndIntegrationAsAdmin( { isTrustedBankAccountSource: true })
+                const tin = trustedOrganization.tin
+                const [trustedProperty] = await createTestBillingProperty(admin, trustedBillingContext)
+                const [trustedAccount] = await createTestBillingAccount(admin, trustedBillingContext, trustedProperty)
+                const [receipt, attrs] = await createTestBillingReceipt(admin, trustedBillingContext, trustedProperty, trustedAccount, { recipient: createTestRecipient({ tin: tin }) })
+                expect(attrs).not.toHaveProperty('receiver')
```

Comment:
We use to separate "prepare" and "check" blocks of code with newline
```suggestion

                expect(attrs).not.toHaveProperty('receiver')
```

Reply:
Done